include/s3select_functions.h

```diff
@@ -965,13 +973,10 @@ struct _fn_like : public base_function
     base_statement* main_expr = *iter;
     value expr_val = expr->eval();
```

Comment:
there is no need for expr , its not used
expr->eval() is a waste of CPU time

Reply:
correct. Updated